src/test/kotlin/uk/gov/justice/digital/hmpps/subjectaccessrequestworker/services/GeneratePdfServiceRestrictedPatientsTest.kt

```diff
@@ -41,16 +43,19 @@ class GeneratePdfServiceRestrictedPatientsTest {
     val testInput = mapOf(
       "prisonerNumber" to "A1234AA",
       "supportingPrisonDescription" to "HMP Exeter",
```

Comment:
I'm going to remove `supportingPrisonDescription` from restricted patients so worth removing this from here too

Reply:
Done